client/src/css/music.css

```diff
@@ -14,169 +14,169 @@ body {
-  margin-bottom: 20px; /* Space below the buttons */
-  gap: 50px; /* Equal spacing between buttons */
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 1em;
+  margin-bottom: 20px;
 }
 
 /* Playlist Switcher Button Styling */
 .playlistSwitcher button {
-  padding: 10px 20px; /* Consistent padding for uniform button size */
-  font-size: 1.2em; /* Ensure the font size is the same for all buttons */
-  background-color: #6a69ae; /* Purple background */
+  padding: 10px 20px;
+  font-size: clamp(1em, 2.5vw, 1.2em);
+  background-color: #6a69ae;
   color: white;
-  border: 2px solid white; /* Purple border */
-  border-radius: 8px; /* Slightly rounded corners for a squared look */
+  border: 2px solid white;
+  border-radius: 8px;
   cursor: pointer;
-  transition: all 0.3s ease; /* Smooth transition for hover and active effects */
-  text-transform: capitalize; /* Capitalize the first letter of each word */
-  box-shadow: 0 4px 8px #6a69ae; /* Subtle shadow for depth */
-  min-width: 160px; /* Fixed width to make buttons the same size */
-  text-align: center; /* Centers text inside the button */
-  height: 70px; /* Fixed height for uniform button size */
+  transition: all 0.3s ease;
+  text-transform: capitalize;
+  box-shadow: 0 4px 8px #6a69ae;
+  min-width: 160px;
+  text-align: center;
+  height: 50px;
 }
 
-/* Hover effect for Playlist Switcher Buttons */
 .playlistSwitcher button:hover {
-  background-color: rgb(40, 23, 80); /* Darker purple on hover */
-  border-color: #6a69ae; /* Darker border on hover */
-  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.15); /* Slightly bigger shadow on hover */
+  background-color: rgb(40, 23, 80);
+  border-color: #6a69ae;
+  box-shadow: 0 6px 12px rgba(0, 0, 0, 0.15);
 }
 
-/* Active effect for Playlist Switcher Buttons */
 .playlistSwitcher button:active {
-  transform: scale(0.98); /* Shrinks slightly when clicked for a "press" effect */
+  transform: scale(0.98);
+}
+
+/* Range Slider Styling */
+input[type="range"] {
+  background-color: #27ae60;
+  border-radius: 10px;
+  height: 6px;
+  width: 100%;
+  transition: background-color 0.3s ease;
+}
+
+input[type="range"]::-webkit-slider-thumb,
+input[type="range"]::-moz-range-thumb {
+  background-color: #27ae60;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  border: none;
 }
 
-/* Responsive Adjustments for Medium Screens */
+/* Responsive Adjustments */
 @media (max-width: 900px) {
   .component {
-      width: 50%;
+    width: 70%;
   }
 }
 
-/* Responsive Adjustments for Small Screens */
 @media (max-width: 600px) {
   .component {
-      width: 90%;
+    width: 90%;
+    padding: 1em;
   }
 
   .musicCover {
-      width: 60%; /* Adjusted for better scaling */
-      margin-bottom: 10px;
+    max-width: 60%;
   }
 
   .title {
-      font-size: 6em;
+    font-size: 3em;
   }
+}
+.music-page {
+  margin-top: 8em; /* Adjust based on your navbar height */
+}
 
-  
-
-  
+.soundSelector {
```

Comment:
White noise clips out of bonds and the items don't look centered

![image](https://github.com/user-attachments/assets/b80d0aa2-bc3b-4c37-aadc-703877f58b73)

![image](https://github.com/user-attachments/assets/7ceff970-2c4e-4f32-8962-89e0e2b12a8a)